src/components/map/ha-map.ts

```diff
@@ -464,7 +465,7 @@ export class HaMap extends ReactiveElement {
         filter: var(--map-filter);
       }
       .dark .leaflet-bar a {
-        background: var(--card-background-color);
+        background-color: var(--card-background-color, #1c1c1c);
```

Comment:
Not sure why but this still doesnt work to change the zoom controls background

Reply:
because `card-background-color` is set, so it will never use `#1c1c1c`

Reply:
It shows as not set though... Weird